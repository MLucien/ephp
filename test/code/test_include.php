<?php

include __DIR__ . "/test_print_true.php" ;
include(__DIR__ . "/test_print_true.php");

include_once __DIR__ . "/test_print_true.php";
include_once(__DIR__ . "/test_print_true.php");


