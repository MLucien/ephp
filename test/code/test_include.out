1

1

Warning: include_once({{CWD}}/test/code/no_exists_file): failed to open stream: No such file or directory in {{CWD}}/test/code/test_include.php on line 4

Warning: include_once(): Failed opening '{{CWD}}/test/code/no_exists_file' for inclusion (include_path='.:') in {{CWD}}/test/code/test_include.php on line 4
1

1

Warning: include({{CWD}}/test/code/no_exists_file): failed to open stream: No such file or directory in {{CWD}}/test/code/test_include.php on line 8

Warning: include(): Failed opening '{{CWD}}/test/code/no_exists_file' for inclusion (include_path='.:') in {{CWD}}/test/code/test_include.php on line 8
1

1
